subgraphs/spotify/src/@types/graphql.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module '*.graphql' {
```

Comment:
Since you switched to `readFileSync` to get the type defs, you might not need this anymore if you're not importing `.graphql` files.